src/main/java/io/excaliburfrc/robot/subsystems/Drive.java

```diff
@@ -45,15 +47,24 @@
 
   private final DifferentialDrive drive = new DifferentialDrive(leftLeader, rightLeader);
 
-  private final DifferentialDriveKinematics driveKinematics =
-      new DifferentialDriveKinematics(DrivetrainConstants.TRACKWIDTH_METERS);
-
   private final DifferentialDriveOdometry odometry;
   private final AHRS gyro = new AHRS();
   private final SparkMaxPIDController leftController = leftLeader.getPIDController();
   private final SparkMaxPIDController rightController = rightLeader.getPIDController();
   private final SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(kS, kV, kA);
 
+  private final DifferentialDriveKinematics driveKinematics =
+      new DifferentialDriveKinematics(DrivetrainConstants.TRACKWIDTH_METERS);
+  private final DifferentialDriveVoltageConstraint voltageConstraint =
+      new DifferentialDriveVoltageConstraint(feedforward, driveKinematics, 10);
+  public final TrajectoryConfig trajectoryConfig =
+      new TrajectoryConfig(MAX_VELOCITY, MAX_ACCELERATION)
+          .setKinematics(driveKinematics)
+          .addConstraint(voltageConstraint);
+
+  public final Vision vision = new Vision();
+  public final Intake intake = new Intake();
```

Comment:
we don't want hard dependencies between subsystems, and this would cause the intake to be allocated twice (=error). Use suppliers etc, and in some cases an external command class.